tests/conformance/scripts/run-tests/runners/index.ts

```diff
@@ -1,21 +1,19 @@
 import CppStreamedTestRunner from "./CppStreamedTestRunner";
 import GoStreamedTestRunner from "./GoStreamedTestRunner";
-import ITestRunner from "./ITestRunner";
 import PythonStreamedReaderTestRunner from "./PythonStreamedReaderTestRunner";
 import PythonStreamedWriterTestRunner from "./PythonStreamedWriterTestRunner";
 import TypescriptIndexedTestRunner from "./TypescriptIndexedTestRunner";
 import TypescriptStreamedReaderTestRunner from "./TypescriptStreamedReaderTestRunner";
 import TypescriptStreamedWriterTestRunner from "./TypescriptStreamedWriterTestRunner";
 
-export type { ITestRunner };
-
-const runners: readonly ITestRunner[] = [
+const runners = [
   new CppStreamedTestRunner(),
   new GoStreamedTestRunner(),
   new PythonStreamedReaderTestRunner(),
   new PythonStreamedWriterTestRunner(),
   new TypescriptIndexedTestRunner(),
   new TypescriptStreamedReaderTestRunner(),
   new TypescriptStreamedWriterTestRunner(),
-];
+] as const;
```

Comment:
Rather than as const I think it would be better to annotate `const runners: readonly (ReadTestRunner|WriteTestRunner)[]` so that the specific types are not exposed to anyone importing this file.